api/src/main/java/org/openmrs/PersonAttributeType.java

```diff
@@ -35,7 +35,7 @@ public class PersonAttributeType extends BaseChangeableOpenmrsMetadata implement
 	
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	@Column(name = "person_attribute_type_id")
+	@Column(name = "person_attribute_type_id", nullable = false)
```

Comment:
do we need nullable = false on an id? Isn't it automatically implied?

Reply:
i have gotten rid of it